E/Enzyme/build_tarballs.jl

```diff
@@ -32,8 +32,9 @@ CMAKE_FLAGS+=(-DLLVM_LINK_LLVM_DYLIB=ON)
 # Build the library
 CMAKE_FLAGS+=(-DBUILD_SHARED_LIBS=ON)
 if [[ "${target}" == *apple* ]]; then
```

Comment:
I'd do
```suggestion
if [[ "${target}" == x86_64-apple* ]]; then
```
because for aarch64 we use 11.0, no need to downgrade it.